lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -392,20 +409,37 @@ def dismiss_welcome_message(request):
 def save_course_goal(request):
     course_id = request.data.get('course_id', None)
     goal_key = request.data.get('goal_key', None)
+    days_per_week = request.data.get('days_per_week', None)
+    subscribed_to_goal_reminders = request.data.get('subscribed_to_goal_reminders', None)
```

Comment:
`None` is already the default value for `get` yeah?

Reply:
good point, removed